src/VisualStudio/CSharp/Impl/LanguageService/CSharpOptionPageService.cs

```diff
@@ -0,0 +1,35 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System;
+using System.Composition;
+using Microsoft.CodeAnalysis.Editor.Implementation;
+using Microsoft.CodeAnalysis.Editor.Shared.Utilities;
+using Microsoft.CodeAnalysis.Host.Mef;
+using Microsoft.VisualStudio.LanguageServices.CSharp.Options.Formatting;
+using Microsoft.VisualStudio.Shell;
+using Microsoft.VisualStudio.Shell.Interop;
+
+namespace Microsoft.VisualStudio.LanguageServices.CSharp.LanguageService
+{
+    [ExportWorkspaceService(typeof(IOptionPageService)), Shared]
+    internal class CSharpOptionPageService : ForegroundThreadAffinitizedObject, IOptionPageService
```

Comment:
this won't work.  You made this an IWorkspaceService.  Which means that if you have this for VB as well the system will have no idea which to load.

Reply:
fixed